lightning/src/ln/channelmanager.rs

```diff
@@ -1341,9 +1341,13 @@ const CHECK_CLTV_EXPIRY_SANITY_2: u32 = MIN_CLTV_EXPIRY_DELTA as u32 - LATENCY_G
 /// The number of ticks of [`ChannelManager::timer_tick_occurred`] until expiry of incomplete MPPs
 pub(crate) const MPP_TIMEOUT_TICKS: u8 = 3;
 
+/// The number of ticks of [`ChannelManager::timer_tick_occurred`] until an invoice request without
+/// a response is timed out.
+pub(crate) const INVOICE_REQUEST_TIMEOUT_TICKS: u8 = 3;
```

Comment:
This is only used in `outbound_payment` so it could live there

Reply:
Moved, as well as `IDEMPOTENCY_TIMEOUT_TICKS`.